languages/ruby/exercises/concept/ostruct/boutique_inventory.rb

```diff
@@ -0,0 +1,23 @@
+class BoutiqueInventory
+  attr_reader :items
+
+  def initialize(items)
+    raise NotImplementError "Please refactor this code so that items have methods"
+
+    @items = items
+  end
+  
+  def item_names
+    raise NotImplementError "Please refactor the code in item_names"
```

Comment:
I like how you have existing code but also the exception.

Reply:
Oh yeah, I do too, makes it easy to "drop in and refactor", and where to do so.